Frontend/ui-driver/src/Components/RideActionModal/View.purs

```diff
@@ -894,6 +895,33 @@ normalRideInfoView push config =
           <> FontStyle.body1 TypoGraphy
       ]
   ]
+  where
+    currency :: String
+    currency = getCurrency appConfig
+
+    parkingChargesView :: forall w . PrestoDOM (Effect Unit) w
+    parkingChargesView = 
+      linearLayout
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , margin $ MarginTop 12
+      , visibility $ boolToVisibility $ Maybe.isJust config.parkingCharge
```

Comment:
@slow-codex  should we have check for specialzone ac pick also ?

Reply:
@max-keviv why should we add such static checks in frontend for parking charges? Where will we get the parking charge value for that and if it's not coming from backend how will it be added in the final fare?

Reply:
and consider it for other cities as well. So if parking charges are not coming then don't show it. Is it handled that way?

Reply:
@witcher-shailesh For now, kept a check only if it's specialZone and YatriSathi. Should this be available to every city?